creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -856,11 +881,12 @@ class SnapbackSM {
    * - For every user on an unhealthy replica, issues an updateReplicaSet op to cycle them off
    * - For every (primary) user on a healthy secondary replica, issues SyncRequest op to secondary
    */
-  async processStateMachineOperation() {
+  async processStateMachineOperation(jobId) {
     // Record all stages of this function along with associated information for use in logging
     const decisionTree = []
     this._addToStateMachineQueueDecisionTree(
       decisionTree,
+      jobId,
```

Comment:
supernit - it would prob make decisiontree needlessly verbose to add `jobId: x` to every stage when you just need it once
thats why in my initial PR i just put it in the data object for the `BEGIN` stage
nbd tho, your call

Reply:
ah nvm, i see you did this bc it pairs with the individual stage logging. got it 👍